misc/pytorch_toolkit/text_recognition/vocabs/vocab_alphanumeric.json

```diff
@@ -0,0 +1,86 @@
+{
+    "id2sign": {
+        "0": "<s>",
+        "2": "</s>",
+        "1": "<pad>",
+        "3": "<unk>",
+        "4": "0",
+        "5": "1",
+        "6": "2",
+        "7": "3",
+        "8": "4",
+        "9": "5",
+        "10": "6",
+        "11": "7",
+        "12": "8",
+        "13": "9",
+        "14": "a",
+        "15": "b",
+        "16": "c",
+        "17": "d",
+        "18": "e",
+        "19": "f",
+        "20": "g",
+        "21": "h",
+        "22": "i",
+        "23": "j",
+        "24": "k",
+        "25": "l",
+        "26": "m",
+        "27": "n",
+        "28": "o",
+        "29": "p",
+        "30": "q",
+        "31": "r",
+        "32": "s",
+        "33": "t",
+        "34": "u",
+        "35": "v",
+        "36": "w",
+        "37": "x",
+        "38": "y",
+        "39": "z"
+    },
+    "sign2id": {
+        "<s>": 0,
+        "</s>": 2,
+        "<pad>": 1,
```

Comment:
I guess you will get rid of \<pad\> symbol eventually?

Reply:
Yes, but for current models this token is needed (to be able to load them with the correct number of classes). Otherwise, I would have to store some bool parameter to add this class for old models or some similar workaround.